notebooks/en/fine_tune_chatbot_docs_synthetic.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p>Maybe <code>Proceed to model loading and fine-tuning.</code> should be bolded </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>